client/src/main/java/gui/playing/PlayBoardController.java

```diff
@@ -167,6 +184,17 @@ private void putFieldOnOpponentBoard(Field field) {
   private final EventHandler<YouHitEvent> youHit =
       event -> lastField.hit();
 
+  private final EventHandler<SunkShipEvent> shipSunk =
+      event -> {
+        String[] positionsAsString = event.getMessage().split(",");
```

Comment:
Maybe you should extract "," as a constant?

Reply:
done